commands/economy/inventory.js

```diff
@@ -25,12 +25,13 @@ module.exports = {
               let inventoryDataProductsIndex = 0
               inventoryDataProducts.forEach((productId) => {
                 const product = inventoryData[productId]
-                fetchShopProduct(client, interaction.guild, productId, (productData) => {
+                getShopProduct(client, interaction.guild, productId, (productData) => {
                   inventoryDataProductsIndex++
                   if (productData && product !== -1) inventoryString += `${productData.productName || 'Non existent item'} - ${productData.productId} (x${product})\n`
                   if (inventoryDataProductsIndex === inventoryDataProducts.length) {
+                    if (!inventoryString) inventoryEmbed.setImage('https://cdn.discordapp.com/attachments/908413370665938975/927155718115688488/empty_inventory.png')
+                    else inventoryEmbed.setThumbnail('https://cdn.discordapp.com/attachments/908413370665938975/917086976744767498/inventory_chest.png')
```

Comment:
Codacy has a fix for the issue: [Expected { after 'else'.](https://app.codacy.com/gh/alexandrumuntenas/pingu/pullRequest?prid=8657799)
```suggestion
                    else {inventoryEmbed.setThumbnail('https://cdn.discordapp.com/attachments/908413370665938975/917086976744767498/inventory_chest.png')}
```